R/derive_param_extreme_event.R

```diff
@@ -89,8 +89,8 @@
 #'
 derive_param_first_event <- function(dataset,
                                      dataset_adsl,
-                                     dataset_source,
-                                     filter_source,
+                                     dataset_add,
+                                     filter_add,
```

Comment:
We are going to need a deprecation warning message telling users to update accordingly - https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#parameter

Reply:
I'm not recalling functions where we have deprecated specific arguments off the top of my head - @pharmaverse/admiral does any remember a function where we did this sothat @PoojaKumari05 can search through GitHub change history for an example??

Reply:
Apologies these comments apply to  `derive_param_extreme_event` as `derive_param_first_event` is deprecated.  You actually don't need to change anything for `derive_param_first_event` as no one can use it.

Reply:
We renamed some arguments in admiral 0.3.0. But this was a long time ago. So maybe just follow the programming strategy?